docs/patterns/components/LoaderOverlay/index.js

```diff
@@ -33,6 +33,9 @@ const LoaderOverlayDemo = () => {
   return (
     <Demo
       component={LoaderOverlay}
+      componentDescription={
+        'A wrapper in which you can mount a Loader component along with some text. It will render a centered, absolutely poisitioned loader box within its relative parent.'
```

Comment:
```suggestion
        'A wrapper in which you can mount a Loader component along with some text. It will render a centered, absolutely positioned loader box within its relative parent.'
```